java/feathub-connectors/flink-connectors/flink-connector-prometheus/src/test/java/com/alibaba/feathub/flink/connectors/prometheus/PrometheusSinkITTest.java

```diff
@@ -0,0 +1,454 @@
+                                                .withComment(
+                                                        "{\"metric_name\":\"test_gauge\",\"type\":\"int\"}")
+                                                .column("bigint_val", DataTypes.BIGINT())
+                                                .withComment(
+                                                        "{\"metric_name\":\"test_gauge\",\"type\":\"bigint\"}")
+                                                .build())
+                                .option("hostUrl", hostUrl)
+                                .option("deleteOnShutdown", "false")
+                                .option("jobName", "testSink")
+                                .option("groupingKey", "k1=v1;k2=v2")
+                                .build())
+                .await();
+
+        final List<Metric> metrics = getMetrics(hostUrl).get("test_gauge");
+        assertThat(metrics)
+                .anyMatch(
+                        metric ->
+                                metric.getLabels().get("type").equals("\"int\"")
+                                        && metric.getValue().equals("99"));
+        assertThat(metrics)
+                .anyMatch(
+                        metric ->
+                                metric.getLabels().get("type").equals("\"bigint\"")
+                                        && metric.getValue().equals("99"));
+    }
+
+    @Test
+    void testSinkColumnWithComment() throws ExecutionException, InterruptedException, IOException {
+        final Table sourceTable =
+                tEnv.fromDataStream(env.fromElements(Row.of(0), Row.of(99))).as("metric_val");
+
+        tEnv.createTemporaryView("sourceTable", sourceTable);
+
+        final Table metricTable =
+                tEnv.sqlQuery(
+                        "SELECT CAST(metric_val AS TINYINT) AS tinyint_val, "
+                                + "CAST(metric_val AS SMALLINT) AS smallint_val, "
+                                + "CAST(metric_val AS INTEGER) AS int_val, "
+                                + "CAST(metric_val AS BIGINT) AS bigint_val, "
+                                + "CAST(metric_val AS DECIMAL) AS decimal_val, "
+                                + "CAST(metric_val AS FLOAT) AS float_val, "
+                                + "CAST(metric_val AS DOUBLE) AS double_val FROM sourceTable");
+
+        final String hostUrl = PROMETHEUS_PUSH_GATEWAY_CONTAINER.getHostUrl();
+        metricTable
+                .executeInsert(
+                        TableDescriptor.forConnector("prometheus")
+                                .schema(
+                                        Schema.newBuilder()
+                                                .column("tinyint_val", DataTypes.TINYINT())
+                                                .withComment(
+                                                        "{\"metric_name\":\"tinyint_metric\",\"value_type\":\"tinyint\"}")
```

Comment:
According to the json spec[1],  string are delimited with double quotation marks and support a backslash [escaping](https://en.wikipedia.org/wiki/Escape_character) syntax.

If we want to make it simpler, we have to parse the configuration ourself, which is what I try to avoid.

[1] https://en.wikipedia.org/wiki/JSON